docs/content/quickstart/tidb-tutorial.md

```diff
@@ -183,12 +183,13 @@ Flink SQL> CREATE TABLE products (
   );
 
 Flink SQL> CREATE TABLE orders (
-   id INT,
-   order_date DATE,
-   purchaser INT,
-   quantity INT,
+   order_id INT,
+   order_date TIMESTAMP(3),
+   customer_name STRING,
+   price DECIMAL(10, 5),
    product_id INT,
-   PRIMARY KEY (id) NOT ENFORCED
+   order_status BOOLEAN,
+   PRIMARY KEY (order_id) NOT ENFORCED
```

Comment:
I think some options in the with clause is invalid, please remove it and make sure the tutorial works well.